body.tex

```diff
@@ -1,215 +1,336 @@
+\end{itemize}
 
+Test Specifications and Verification Baseline documents are create for specific components, where components are the main products in a subsystem.
 
-\section{Information Summary and Control Document}
 
-As it has been described in the above section, all Rubin Observatory test information is managed in Jira. 
-This permits to extract very easily the Verification Control Document (VCD), that shows the level of coverage for each requirement.
+\section{The Verification Control Document}
 
-The VCD is a latex document generated using the document generation tool Docsteady. 
-It is managed in the same way as the Test Plan and Report, the latex code is managed in Github,
-built in Travis and published in lsst.io.
+As mentioned above, all Rubin Observatory test information is managed in Jira.
+This makes it easy to extract the Verification Control Document (VCD), where we summarize the level of test coverage of requirements.
 
-Since the number of requirement for the Rubin Observatory is very high, the VCD is generated per subsystem.
+The VCD is a \LaTeX document generated using the document generation tool Docsteady.
+As for other documents the VCD is managed in a GitHub repository, built with Travis CI and published through LSST the Docs.
 
-The VCD provides 2 main sections:
+Since the number of requirement and verification elements for the Rubin Observatory is very high --
+approximately 700 requirements and 1000 verification elements in DM alone -- a separate VCD is generated for each subsystem.
+In the document there are 2 main sections:
 
 \begin{itemize}
-\item \textbf{Summary Information} where a status overview is given. 
-This includes the number of requirements and verification elements related to passed or failed test cases.
-\item \textbf{Detailed Information} where for each requirement it is shown which are the verification elements and test cases
-and the status of the test cases. Links to the test documents are provided, but not descriptive information.
+\item \textbf{Summary Information} where a status overview is given.
+This includes the number of requirements and verification elements covered by passed test cases.
+Figure \ref{fig:vcdsum} shows an example extracted from the DM VCD.
+\item \textbf{Detailed Information} where for each requirement, the verification elements, test cases
+and the status of the test cases are shown. Links to the test documents are provided, but not descriptive information.
+Figure \ref{fig:vcddetail} shows a detail example extracted from the DM VCD.
 \end{itemize}
 
-The VCD become an important document for management to know the level of verification and validation that has been achieved so far.
-At the same time it can be provided to reviews in order to demonstrate that the expected milestones have been met.
-
+The VCD is an important management tool assessing the  level of verification and validation that has been achieved so far.
```

Comment:
The VCD is an important management tool for assessing the  level of verification and validation that has been achieved to date.